tests/unit/blocking_connection_tests.py

```diff
@@ -205,6 +210,46 @@ def test_sleep(self, select_connection_class_mock):
                 spec_set=blocking_connection.BlockingConnection._flush_output):
             connection.sleep(0.00001)
 
+    def test_client_properties_default(self):
+        expectation = {
+            'product': connection.PRODUCT,
+            'platform': 'Python %s' % platform.python_version(),
+            'capabilities': {
+                'authentication_failure_close': True,
+                'basic.nack': True,
+                'connection.blocked': True,
+                'consumer_cancel_notify': True,
+                'publisher_confirms': True
+            },
+            'information': 'See http://pika.rtfd.org',
+            'version': pika.__version__
+        }
+        with mock.patch('pika.connection.Connection.connect'):
+            with self.assertRaises(exceptions.ConnectionClosed):
+                conn = blocking_connection.BlockingConnection()
+                self.assertDictEqual(conn._client_properties, expectation)
```

Comment:
BlockingConnection doesn't have the `_client_properties` member, since it's not subclassed from the `Connection` hierarchy. The Connection hierarchy is reachable via `conn._impl`.